### PR TITLE
update GC-MS deconvolution file

### PR DESCRIPTION
**Pull request by MelissaESPOSITO**

### CODE REVIEW

**Comment by MelissaESPOSITO** [comment]
@mwang87 it's ready to merge! thanks

File: docs/gc-ms-deconvolution.md [hunk 1]
```diff
@@ -33,11 +33,18 @@ The GNPS home page includes two sections to launch GNPS GC-MS data analysis job,
...
 
 
 
 The **“Advanced MSHub Processing” section** displays some automatic setting parameters (related to peak symmetry and baseline adjustment) determined internally by MSHub. The user does not need to provide settings, although they can be set manually, should the user choose to do so. 
 
+The adjustable settings are:
```
Comment (mwang87) [inline docs/gc-ms-deconvolution.md:42]: Do you know if we actually use these? I thought I disabled them in the actual code.

Comment (MelissaESPOSITO) [reply docs/gc-ms-deconvolution.md:42]: you're right, the user cannot choose to change parameters for Ms-Hub, but Alexander asked me to add that so I was thinking that you planned to add this option.

Comment (MelissaESPOSITO) [reply docs/gc-ms-deconvolution.md:42]: maybe a reviewer asked for that, I don't have the explaination but I can ask to Alexander.

**Comment by MelissaESPOSITO** [comment]
@mwang87 
It's ok to merge. Let me know if you see some mistakes.

**Comment by MelissaESPOSITO** [comment]
Alex told me that it's possible to change "auto" by a number.

**Review by lfnothias — APPROVED** [review]
It looks good.

**Pull request closed without merging**